src/typelib.erl

```diff
@@ -48,7 +48,11 @@ pp_type(Type = {type, _, bounded_fun, _}) ->
     "fun(" ++ S ++ ")";
 pp_type({var, _, TyVar}) ->
     %% TODO: In type(), TyVar should be an atom but we use a string.
-    TyVar;
+    %% EDIT: Apparently, not everywhere :/
```

Comment:
For explicit type variables, it's an atom. For implicit ones (auto-generated for fun parameters), it's a string. Perhaps it's good to describe that, before we forget it again. :-)

Reply:
Yeah, this was not very consistent...
https://github.com/josefs/Gradualizer/commit/0e898384d640363efb13cd30e8d4bd472df4bec9#diff-c46e88595065283f0502281ada54f0a29daeff24290d35ad89c44161760b2753R263-R265

But your comment clears it up :+1: